yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,512 @@
+<p style="padding-bottom: 7px"></p>
+
+## Define the Intent Handler for *makeGuessIntent* (Part 2)
+Now we want our **makeGuessIntent** to respond differently based on our **myGuessSlot** value. **IF** our guess is greater than our **secret** variable, **THEN** Alexa will **ask**: “(myGuessSlot) is too big. Guess again”. The user will then have another opportunity to guess, which will create a loop until the user guesses correctly. **IF** our guess is less than our **secret** variable, **THEN** Alexa will **ask**: “(myGuessSlot) is too small. Guess again”, which will once again create a loop. **IF** our guess is accurate, **THEN** Alexa will say: “<myGuessSlot> is correct. Congratulations!"
+
+Because we want a different response based on our **myGuessSlot** value, we need to use **conditionals**. **Conditionals** provide us with the ability to choose different responses based on different parameters using **if/else if/then** statements. Let’s try it out!
+
+To start, open the **Control** drawer and drag out the <span class = "control">if/then/else if/then/else</span> block, and place it into the <span class = "control">when makeGuessIntent.spoken</span> block.
+
+![Control Drawer](../images/number_guessing_game/image10.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+![Add if... statement](../images/number_guessing_game/image4.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Let's first check whether our **myGuessSlot** value is greater than our secret variable. To do so, open the **Math** drawer and pull out the <span class = "math">equality</span> function block and attach it to the "if" statement.
+
+![Math Equality](../images/number_guessing_game/image40.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Next, click the equal sign, and in the drop-down menu, pick the greater than sign (>). 
+
+![Math Inequality](../images/number_guessing_game/image39.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Let's fill in these empty values! In the first space, put in **myGuessSlot.value** from the **myGuessSlot** drawer and in the second space, put in <span class = "variables"> get global secret</span> from the **Variables** drawer. 
+
+![Insert values](../images/number_guessing_game/image50.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+We've now set our first if statement! **IF** our **myGuessSlot** value is greater than our secret value, then the expression within the first **then** statement will be run. Let's define this first **then** statement - we want Alexa to **ask**: "(myGuessSlot) is too big. Guess again". To do this, we need to once again use the **ask** block from the **Voice** drawer, the **myGuessSlot.value** block, and the <span class = "text">join</span> block from the **Text** drawer. Try it yourself! (Hint: How many parts are there in Alexa's response? Do we need to add parts to the **join** block? Click the hint to check your blocks!)
```

Comment:
not sure what is the intended formatting for (myGuessSlot). Fix everywhere please.